packages/client/wallets/aa/src/types/Config.ts

```diff
@@ -30,11 +25,6 @@ export type ViemAccount = {
 
 export type PasskeySigner = {
     type: "PASSKEY";
-
-    /**
-     * Displayed to the user during passkey registration or signing prompts.
-     */
-    passkeyName: string;
```

Comment:
I think we should still be able to specify this, with the userId derived from the jwt acting as the default. That said, I always like having less stuff, wdyt?

Reply:
If we do this, then I believe we have to switch the whole approach to obtain the userId client side instead of server side, right? Something like a pre step os checking if no passkeyName was submitted, then parse the jwt, check the jwtConfig of the api key (not sure we expose that now) to see what we use as a claim, and then proceed with that. I guess we can always do that later if we want to and start this way? No strong opinion tbh, just that one because the work is already done haha

Reply:
Mmmm maybe there is another option. We could expose an optional passkeyName here where you commented, and internally given that ZD sends a required passkeyName (the `no-op` string I chose) If null we can send an empty string (or the no-op string) as an indicator to the backend to infer the passkey name to the jwt. And we don't have to port that logic to the client. 
How does that sound?

Reply:
perfect